### ISSUE TITLE
[Syllabus Map]

### ISSUE THREAD

**Issue by neenjaw** [exercism/exercism#5847]
I'd like to give more control to authors how their syllabus map appears. Take for example this layout on the JavaScript track, suppose the author wanted to order them alphabetically. 

![Screenshot from 2021-09-08 20-52-26](https://user-images.githubusercontent.com/8953691/132615066-806a906e-99d1-4d2f-a54c-1f3c7e0ad3ae.png)

A simple solve would be to allow authors to reorder their concepts by changing their ordering in the config.json.  If this order were persisted to the concept record, then it could be used when determining the syllabus map layout.

I'm happy to PR the changes to the syllabus map once this is complete.

I would expect this to be low priority at this time, but I'm logging it here for some time in the future.

**Comment by iHiD** [comment]
@ErikSchierboom You should be able to duplicate the syncing code from exercises for this one.
